### ISSUE TITLE
Raise version of distributeme-core (2.5.1) that does not depend on outdated jackson-mapper-asl 1.9.2

### ISSUE THREAD

**Issue by in-fke** [anotheria/moskito#265]
moskito depend on distributeme in an outdated version that again depends on outdated jackson-mapper-asl 1.9.2 which is bad.
jacskon-mapper-asl is really outdated, see https://mvnrepository.com/artifact/org.codehaus.jackson/jackson-mapper-asl

Can distributeme be raised from 2.3.10 to 2.5.1 without the need of making any code changes in moskito?
Or, as a temporary workaround: could jackson-mapper-asl be excluded from the dependency tree? when using moskito 3.2.2 ?

```
[INFO] +- net.anotheria:moskito-inspect-embedded:jar:3.2.2:compile
[INFO] |  \- net.anotheria:moskito-webui:jar:3.2.2:compile
[INFO] |     +- net.anotheria:ano-maf:jar:3.0.0:compile
[INFO] |     +- net.anotheria:ano-plass:jar:2.1.4:compile
[INFO] |     +- net.anotheria:ano-tags:jar:2.1.1:compile
[INFO] |     |  \- commons-el:commons-el:jar:1.0:compile
[INFO] |     +- jstl:jstl:jar:1.2:compile
[INFO] |     +- net.anotheria:ano-net:jar:2.1.0:compile
[INFO] |     +- net.anotheria:distributeme-core:jar:2.3.10:compile
[INFO] |     |  \- org.codehaus.jackson:jackson-mapper-asl:jar:1.9.2:compile
[INFO] |     |     \- org.codehaus.jackson:jackson-core-asl:jar:1.9.2:compile
[INFO] |     +- net.anotheria:distributeme-generator:jar:2.3.10:compile
[INFO] |     +- net.anotheria:distributeme-support:jar:2.3.10:compile
[INFO] |     +- net.anotheria:ano-prise:jar:2.2.1:compile
[INFO] |     \- net.sf.dozer:dozer:jar:5.1:compile
```

**Comment by in-fke** [comment]
Also, upgrading distributeme would drop transitive dependency to artifact "jsr311-api", which usually conflicts with artifacts that specify "JAX-RS 2.x".

**Comment by dvayanu** [comment]
Fixed and released as 3.2.5
